charts/azuremonitor-containers/values.yaml

```diff
@@ -19,7 +19,7 @@ Azure:
     httpsProxy: ""
     noProxy: ""
     proxyCert: ""
-omsagent:
+amalogs:
   image:
```

Comment:
what's the reason to have some places "amalogs" and other places "ama-logs"?

Reply:
values file cannot handle "-" in keys due to parsing issues